web/modules/custom/asset_distribution/config/install/field.field.rdf_entity.asset_distribution.field_ad_licence.yml

```diff
@@ -9,7 +9,7 @@ field_name: field_ad_licence
 entity_type: rdf_entity
 bundle: asset_distribution
 label: License
-description: ''
+description: 'If you cannot find the desired license in the list above, please request a new one by <a href="/contact">contacting us</a>.'
```

Comment:
@claudiu-cristea, Thank you for pointing this out. I added the site path to the link. 
For the moment we cannot follow the link to contact page in the tests because the page doesn't exist. I created a followup ticket for the contact form.